engines/bastion_katello/app/assets/stylesheets/bastion_katello/bastion_katello.scss

```diff
@@ -26,4 +26,7 @@
     text-overflow: ellipsis;
   }
 
+  .breadcrumb {
+    margin-bottom: 0px;
```

Comment:
I am seeing 8px of padding on the top and bottom of a breadcrumb (looks like this is coming from bootstrap). Do we really want to add another 10px to that?
![image](https://cloud.githubusercontent.com/assets/5964462/17447130/ca468b38-5b1a-11e6-8398-1fbccbaf5584.png)